src/webgpu/shader/validation/wgsl/runtime-array-not-last-v3.fail.wgsl

```diff
@@ -0,0 +1,8 @@
+# v-0015 - This fails because of the runtime array is not last member of the struct.
```

Comment:
This is a weird error message for this case since there's no struct. I think it could simply say "of a struct" or "of a [[block]] struct" at the end and that would be ok.

Reply:
Yes, that's right! fixed.